lib/analysis/author/community.py

```diff
@@ -0,0 +1,265 @@
+"""
+This module is used to find the community structure of the network according to the Infomap method of Martin Rosvall
+and Carl T. Bergstrom and returns an appropriate VertexClustering object. This module has been implemented using both
+the iGraph package and the Infomap tool from MapEquation.org. The VertexClustering object represents the clustering of
+the vertex set of a graph and also provides some methods for getting the subgraph corresponding to a cluster and such.
+
+"""
+import json
+import subprocess
+import sys
+
+import networkx as nx
+import igraph
+import numpy
+# import plotly
+# from matplotlib import pyplot as plt
+# from plotly.tools import FigureFactory as FF
+# from scipy.cluster.hierarchy import dendrogram, linkage
+
+from analysis.author import ranking
+from util.read import *
+
+sys.setrecursionlimit(10000)
+
+
+def write_matrix(json_data, tree_filename="infomap/output/" + "author_graph.tree"):
+    """
```

Comment:
sphinx docs please

Reply:
I had added Sphinx docs during the last review. I am surprised they did not carry over!